next-app/src/routes/projects/[project_code]/activities/+server.ts

```diff
@@ -66,12 +71,13 @@ export async function fetch_activities({ cookie, start_date, end_date }: Activit
 	return activity.map(transform)
 }
 
-function transform({ id, action, date, content }: LegacyActivity): Activity {
+function transform({ id, action, date, content, userRef }: LegacyActivity): Activity {
```

Comment:
is there an intention to use these somewhere?

Reply:
These variables are used in the project activity logs. I might not be understanding your question, let me know if you are clarifying something different